src/visual_mode/visual_assembler.md

```diff
@@ -1,7 +1,8 @@
 # Visual Assembler
 
-You can use Visual Mode to assemble code using `A`.
-For example let's replace the `push` by a `jmp`:
+You can use Visual Mode to assemble code (patch) using `A`.
+For example, let's `xor` the EAX register instead of EBP register, here.
+To assemble, seek to the location you want to patch and press `A`.
```

Comment:
In the next paragraph, it says: "Notice the preview of the disassembly and arrows:"

What arrows?

Reply:
I think it was meant to say jump/call reference lines. Which are obviously missing on the picture, probably worth to change the example to show the one with changing jumps, e.g. something like from `jz` to `jnz` or similar.

Reply:
What about this one?
![before](https://user-images.githubusercontent.com/29057155/119341624-a34e8f00-bcb1-11eb-895c-b63438e1dcf5.png)
![after](https://user-images.githubusercontent.com/29057155/119341619-a184cb80-bcb1-11eb-9e9d-5470391af581.png)